### PR TITLE
MukherjeeHW01

### PR DESCRIPTION
**Pull request by deblnia**
I have completed my homework!

### CODE REVIEW

**Comment by mayank-kgp** [comment]
@deblnia I forgot to tell you that you need to write your brief biography in the readme file using markdown. Please read the sections  **Practice using markdown** and **Report your process** at https://cfss.uchicago.edu/hw01-edit-README.html
Let me know if you have any confusion.

**Comment by mayank-kgp** [comment]
@deblnia You need to include a description of how you got the changes into README.md on GitHub when you submit your assignment via pull request. Please check the "Report your process" section on https://cfss.uchicago.edu/hw01-edit-README.html

**Comment by deblnia** [comment]
I got the changes into README.md on Github using the following series of commands in terminal: 

1. git add [local repo name] 
2. git commit -m "[commit message]" 
3. git push -u origin [local repo name]
4. Once the README.md was added to my Github, I opened a pull request on the main branch. 

Although I was able to submit the assignment with lots of help of Mayank, I'm still unclear about a lot of Github's operating functions. For example, I can't remember if I used the command written or just simply "git push". Further, when I needed to update the pull request to fully report my process I wasn't sure if I should re-commit and push the project again. I found the cheatsheet and [this](https://services.github.com/on-demand/github-cli/open-pull-request-github) helpful in figuring out terminal commands, and I'm sure that this will become more procedural as the quarter progresses. 

Oh, also, on actually completing the assignment: I just forked the repo using the Github link, created a local copy, opened it in RStudio and started writing in Markdown. The formatting stuff wasn't difficult at all (I mostly struggled with the way Markdown interprets spaces i.e. *italicizes, * creates a list) in RMarkdown.
The most trouble I had was in getting the photograph to render. Some websites couldn't seem to agree on the proper syntax, and then when I found the proper syntax (  ) it took me a while to realize that if I didn't want any caption text I could just leave the brackets blank. I got it eventually.

**Comment by deblnia** [comment]
Oh, also, on actually completing the assignment: I just forked the repo using the Github link, created a local copy, opened it in RStudio and started writing in Markdown. The formatting stuff wasn't difficult at all in Markdown, even though I did have some trouble getting the photograph to render. (I got it eventually.)

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
...
+
+## Research 
+
+I spent the early part of the summer working with Karthick Ramakrishnan at [AAPI Data](aapidata.com), and I did research with [Professor Robert Gulotty](http://home.uchicago.edu/~gulotty/index.html) focusing on political economy in American Rust Belts for the rest of the summer. I've been involved with the [Knowledge Lab](https://www.knowledgelab.org/) since my first year, and currently have a paper under review at [Brown's JPPE](https://www.brownjppe.com/).  
+
+## Comedy 
+
+I do two comedy activities on-campus: 
+
+***1. Occam's Razor Comedy Improv***
```
Comment (bensoltoff) [inline README.md:19]: Another way to format this list:

```
1. Occam's Razor Comedy Improv
    I practice and perform improvisational comedy.
1. The Chicago Shady Dealer
    I both write and compile layout for this publication. 
```

1. Occam's Razor Comedy Improv
    I practice and perform improvisational comedy.
1. The Chicago Shady Dealer
    I both write and compile layout for this publication.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,26 @@
 # hw01
 Repository for homework 01
+
+# Deblina Mukherjee 
+
+I'm a second year in the College at UChicago majoring in **sociology** (with a specialization in *computational social science*) and **statistics**. 
+
+
+![](https://raw.githubusercontent.com/deblnia/deblnia.github/master/pic3.jpg)
```
Comment (bensoltoff) [inline README.md:9]: Alternatively you could store the picture in this repo.

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |    x       |              |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Lots of Markdown elements
* Clean repo - no extraneous files

**Pull request closed without merging**